addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -160,7 +160,7 @@
                 <t t-if="question_count or tags" t-call="website.website_search_box_input">
                     <t t-if="_page_name == 'tags'">
                         <t t-set="search_type" t-value="'forum_tags_only'"/>
-                        <t t-set="action" t-valuef="'/forum/%s/tag' % (_forum_slug)"/>
+                        <t t-set="action" t-valuef="/forum/#{ _forum_slug }/tag"/>
```

Comment:
We certainly can do what you did, but then can we do the same below so that they "look the same"?

Otherwise, we can also just remove the `f` for a even shorter fix that keeps them aligned.